### DISCUSSION TITLE
how to add custom audio sound?

### DISCUSSION THREAD

**Discussion by KaifKol** [YimMenu/YimMenu#3594] [Q&A] [answered]
Is it possible to make it so that when a button is pressed, a certain sound from the scripts folder will be played in .mp3 format?

**Comment by xesdoog** [comment, marked as answer]
No that's not possible. The reason is simply because Lua itself can not play Audio, you need a separate module / library to do so, [see example here](https://luarocks.org/modules/luarocks/luacom).
YimMenu's Lua API does not have such a module. If you want you can fork YimMenu, add LuaCOM to the current Lua API then open a PR although I doubt it will be merged.
